staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/time.go

```diff
@@ -90,7 +90,7 @@ func Unix(sec int64, nsec int64) Time {
 
 // Rfc3339Copy returns a copy of the Time at second-level precision.
 func (t Time) Rfc3339Copy() Time {
-	copied, _ := time.Parse(time.RFC3339, t.Format(time.RFC3339))
+	copied, _ := time.Parse(time.RFC3339, t.UTC().Format(time.RFC3339))
```

Comment:
there are a lot of ways of constructing a new Time instance... it seems odd to make this one coerce to UTC and leave NewTime/Date/Now/Unix/UnmarshalJSON/UnmarshalQueryParameter returning local times

Reply:
as far as wire compatibility, MarshalJSON/ToUnstructured/MarshalQueryParameter already coerce to UTC, so I tend to think that making the New/Unmarshal methods do the same is *probably* a good idea, but I could also imagine that changing in-memory behavior or failing deep-equal comparisons with locally constructed times

Reply:
> as far as wire compatibility, MarshalJSON/ToUnstructured/MarshalQueryParameter already coerce to UTC, so I tend to think that making the New/Unmarshal methods do the same is _probably_ a good idea, but I could also imagine that changing in-memory behavior or failing deep-equal comparisons with locally constructed times

You are right, this UTC is meaningless under time.Parse